components/Banner/examples/tests/snapshot/MDCBannerSnapshotTests.m

```diff
@@ -36,9 +37,13 @@ - (void)setUp {
 
   // Uncomment below to recreate all the goldens (or add the following line to the specific
   // test you wish to recreate the golden for).
-  //  self.recordMode = YES;
+//    self.recordMode = YES;
```

Comment:
Please revert this change - the older formatting was correct.

Reply:
Addressed in https://github.com/material-components/material-components-ios/pull/7018/commits/6db8cc73f124506de79933b0abd74dbdec9ac48b.